dev-requirements.txt

```diff
@@ -1,49 +1,50 @@
 -i https://pypi.python.org/simple
 attrs==21.2.0; python_version >= '2.7' and python_version not in '3.0, 3.1, 3.2, 3.3, 3.4'
-backports.entry-points-selectable==1.1.0; python_version >= '2.7'
-bandit==1.7.0
-certifi==2021.5.30
+backports.entry-points-selectable==1.1.1; python_version >= '2.7'
+bandit==1.7.1
+certifi==2021.10.8
 cfgv==3.3.1; python_full_version >= '3.6.1'
-charset-normalizer==2.0.5; python_version >= '3'
-coverage==6.0b1
+charset-normalizer==2.0.9; python_version >= '3'
+coverage==6.2
 decorator==5.1.0; python_version >= '3.5'
-distlib==0.3.2
+distlib==0.3.4
 execnet==1.9.0; python_version >= '2.7' and python_version not in '3.0, 3.1, 3.2, 3.3, 3.4'
-filelock==3.0.12
-gitdb==4.0.7; python_version >= '3.4'
-gitpython==3.1.18; python_version >= '3.6'
-greenlet==1.1.1; python_version >= '3' and platform_machine in 'x86_64 X86_64 aarch64 AARCH64 ppc64le PPC64LE amd64 AMD64 win32 WIN32'
-hypothesis==6.21.3
-identify==2.2.14; python_full_version >= '3.6.1'
-idna==3.2; python_version >= '3'
+filelock==3.4.0; python_version >= '3.6'
+gitdb==4.0.9; python_version >= '3.6'
+gitpython==3.1.24; python_version >= '3.7'
+greenlet==1.1.2; python_version >= '2.7' and python_version not in '3.0, 3.1, 3.2, 3.3, 3.4'
```

Comment:
I may be misunderstanding how environment markers work, so let me ask a clarifying question here: 
We set `[requires] python_version = "3"` in `Pipfile`, `[mypy] python_version=3.6` in `mypy.ini`, and  `PYPI_CLASSIFIERS` in `setup.py` to be essentially `3.9 => python_version => 3.6`. That leaves us with documented compatibility with Python 3.6-3.9. 
When relocking dependencies, why do we ask dependencies to enable compatibility with (for example) `python_version >= '2.7'` etc. instead of `python_version >= '3.6'?` Is it to maintain backward-compatibility with previous Python 3 versions, perhaps caused by setting `[requires] python_version = "3"` in `Pipfile`?

Reply:
Good question. This is automatically produced by `pipenv` when running the `pipenv lock ... --requirements >  [requirements_file]` command to relock dependencies. There may be ways to refine it, or perhaps this has improved in newer versions of pipenv but haven't looked into it recently.

Maybe https://github.com/pypa/pipenv/pull/4751 which was included in version 2021.11.5 (https://pipenv.pypa.io/en/latest/changelog/#id24) may help (?)